packages/gatsby-theme-api-docs/src/components/type/enum.js

```diff
@@ -23,7 +23,7 @@ const Enum = ({
         {description && (
           <DescriptionParagraph>{description}</DescriptionParagraph>
         )}
-        <Markdown.Dl>
+        <Markdown.DescriptionList>
```

Comment:
As agreed I changed the `<d*>` components name to something more descriptive name

Reply:
How they are named internally in the "Markdown" module does not really matter, what matters is under what name they are injected into the MDX provider.    (e.g. the "Markdown." prefix is never exposed)